mne_bids_pipeline/tests/configs/config_ERP_CORE.py

```diff
@@ -33,13 +33,12 @@
 task = os.environ.get('MNE_BIDS_STUDY_TASK')
 sessions = [task]
 
-subjects = ['015', '016', '017', '018', '019']
-
+subjects = ['015', '016', '017', '018']
```

Comment:
@hoechenberger okay to reduce this to 4? 4 or 6 make more sense than 5 because `N_JOBS=2`, 4 will be faster than 5 but 5 should be about the same amount of execution time as 6.

Reply:
I think I picked 5 because some effects are only visible when averaging across a certain minimum number of participants. We can try 4, but need to look at the grand average reports to ensure that the results still look somewhat "convincing".

Reply:
@larsoner I'm looking into this right now and adjusting parameters. will push to this branch once i have results

Reply:
@hoechenberger it didn't actually affect the run time all that much. So we can also just revert if you want

Reply:
Yes please do -- it's okay on my Linux machine but pretty bad on my M1

Reply:
Any idea why that might be? I also have the impression it used to be faster on my Intel Mac

Reply:
For me it slams all cores even if I set `OPENBLAS_NUM_THREADS=2` for example. I think it might thus be due in part to a parallelization oversubscription, as we run `N_JOBS=2` and I think both try to use all cores... but I haven't fully checked.

Reply:
I found this weird behavior, but I suspect it's a bug in Python itself?

```shell
❯ python -c 'import platform; print(platform.processor())'
i386
❯ arch -arm64 python -c 'import platform; print(platform.processor())'
arm
❯ lipo -archs `which python`
arm64
```
So the binary is clearly ARM only, but when run, Python reports it's running on x86 unless one uses the `arch§ command, which to my knowledge should only be of use with fat binaries…

Reply:
I just ran with `n_jobs` set to 1 and this actually finished faster than with `2`, yet still utilized all CPU cores. Definitely resource oversubscription.

We're running the ERP CORE tests through Dask, if I'm not mistaken. So the problem might lie there…

Reply:
> I found this weird behavior, but I suspect it's a bug in Python itself?

I do not get that on my ARM/M1 machine with conda-forge Python:
```
$ python -c 'import platform; print(platform.processor())'
arm
```
so I suspect something is wrong with your `which python` / nativeness of the CPython you're using.

Reply:
Extremely weird…